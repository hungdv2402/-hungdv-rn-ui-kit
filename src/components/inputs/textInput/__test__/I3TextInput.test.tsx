import renderer from 'react-test-renderer';
import I3TextInput from '../I3TextInput';
import React from 'react';

describe('renders I3TextInput correctly', () => {
  it('should render I3TextInput with label="Email"', () => {
    const tree = renderer
      .create(
        <I3TextInput onFocus={jest.fn()} label='Email' />
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});