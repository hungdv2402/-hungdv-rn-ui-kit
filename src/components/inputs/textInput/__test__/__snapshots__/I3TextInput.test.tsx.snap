// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders I3TextInput correctly should render I3TextInput with label="Email" 1`] = `
<View
  style={
    {
      "alignSelf": "stretch",
      "justifyContent": "flex-start",
    }
  }
>
  <Text
    style={
      {
        "fontSize": 14,
        "fontWeight": "400",
        "paddingBottom": 8,
      }
    }
  >
    Email
  </Text>
  <TextInput
    label="Email"
    onBlur={[Function]}
    onFocus={[Function]}
    placeholderTextColor="#B8B9BA"
    style={
      [
        {
          "backgroundColor": "#fff",
          "borderColor": "#D3D3D4",
          "borderRadius": 8,
          "borderWidth": 1,
          "fontSize": 14,
          "fontWeight": "400",
          "minHeight": 53,
          "paddingHorizontal": 16,
        },
        false,
      ]
    }
  />
</View>
`;
